pkgs/development/misc/resholve/resholve-utils.nix

```diff
@@ -181,6 +181,7 @@ rec {
       src = unresholved;
       inherit version pname;
       buildInputs = [ resholve ];
+      disallowedReferences = [ resholve ];
```

Comment:
Setting this here to make sure that we have no runtime references to `resholve` itself.